packages/workbox-core/_private/cacheWrapper.mjs

```diff
@@ -36,6 +36,16 @@ import '../_version.mjs';
  * @memberof module:workbox-core
  */
 const putWrapper = async (cacheName, request, response, plugins = []) => {
+  if (!response) {
+    if (process.env.NODE_ENV !== 'production') {
+      logger.error(`Request '${getFriendlyURL(request.url)}' had no URL.`,
+        responseToCache);
```

Comment:
`responseToCache` is declared later on in the function, so don't use it here.

I think you mean to log `response`, instead. Are you doing that because you want to distinguish between `undefined` and `null`? Do you think that would be a meaningful distinction when debugging?